docs/manufacturing.md

```diff
@@ -1,13 +1,67 @@
 # Babel-Babel manufacturing
 
+## PCB specification for all PCB
+### Main PCB
+* Quantity per product/order: 1
```

Comment:
The quantity specification is redundant because it is already contained in the BOM table below, so it should be removed. You can add a forward reference if desired (like "see below").

This comment also applies to the following sections.